### PR TITLE
Add Cloud Security Engineer to list of roles

### PR DESCRIPTION
**Pull request by ker-an**

### CODE REVIEW

File: src/data.json [hunk 1]
```diff
@@ -309,28 +309,28 @@
                                 "Build good governance - Manage and monitor resources and risks, proactively plan and take corrective action where necessary.",
                                 "Create vision and value - Help to define, shape and prioritise the vision, goals, initiatives and outcomes - seeking value, avoiding complexity.",
                                 "Manage 3rd parties - Manage and collaborate with 3rd parties to ensure they deliver effectively and that contracts are structured appropriately.",
-                                "Build community - Work to improve the FT’s Agile capability. Foster Agile knowledge, the supporting community and tools across the organisation"
+                                "Build community - Work to improve the FT’s Agile capability. Foster Agile knowledge, the supporting community and tools across the organisation."
                             ]
                         },
                         "requirements": {
                             "title": "This role will be right for you if...",
```
Comment (kateloschinina) [inline src/data.json]: We will have to replace ... with "..." sign (as per Alice's PR #61)

File: src/data.json [hunk 2]
```diff
@@ -309,28 +309,28 @@
...
-                                "3+ years leading and coaching Agile teams",
-                                "Strong experience and understanding of digital product development",
-                                "A good understanding of lean/agile principles and multiple methodologies",
-                                "Strong experience of coaching agile teams and individuals, workshop facilitation and structured problem solving",
-                                "Experience managing delivery using measurable business outcomes"
+                                "3+ years leading and coaching Agile teams.",
+                                "Strong experience and understanding of digital product development.",
+                                "A good understanding of lean/agile principles and multiple methodologies.",
+                                "Strong experience of coaching agile teams and individuals, workshop facilitation and structured problem solving.",
+                                "Experience managing delivery using measurable business outcomes."
```
Comment (kateloschinina) [inline src/data.json]: On line 366: "This role is within Enterprise Services (ES)  Networks" - there is a double space there

File: src/data.json [hunk 3]
```diff
@@ -377,27 +377,27 @@
...
+                                "You have professional technical accreditations.",
+                                "You have experience in Python or similar, with the desire to learn new languages.",
                                 "You have experience with Cloud Computing e.g. IaaS, PaaS, SaaS etc.",
-                                "You are familiar with a range of AWS  technologies (such as Cloudformation, Lambda and EC2)",
-                                "You have the ability to work with APIs",
-                                "You have experience with version control systems, e.g. Git",
-                                "You produce effective documentation to communicate anything from an idea to a strategy",
-                                "You have experience defining and implementing testing processes and procedures",
-                                "You have an understanding of agile development and its core principles",
+                                "You are familiar with a range of AWS  technologies (such as Cloudformation, Lambda and EC2).",
```
Comment (kateloschinina) [inline src/data.json]: AWS  technologies - double space

File: src/data.json [hunk 4]
```diff
@@ -511,36 +511,109 @@
...
-                                "You have the  ability to work independently, but with regular support and collaboration with their manager",
-                                "You have the ability to balance workloads and priorities throughout the product lifecycle across of a range of email products and offsite services"
+                                "You have a proven track record in a product management role (2 years +).",
+                                "You are data-driven and hypothesis led with ability to balance alongside user research and business objectives.",
+                                "You are customer obsessed.",
+                                "You have good communication skills – able to clearly write and present documents targeted at various audiences, as well as make presentations to large/small groups.",
+                                "You are proactive, outcome driven, with a can-do attitude.",
+                                "You are intellectually curious,  with a demonstrable ability to identify opportunities and communicate their value proposition and business benefits.",
```
Comment (kateloschinina) [inline src/data.json]: "curious,  with" - double space

File: src/data.json [hunk 5]
```diff
@@ -511,36 +511,109 @@
...
+                                "You have the  ability to work independently, but with regular support and collaboration with their manager.",
```
Comment (kateloschinina) [inline src/data.json]: "the  ability" - double space

File: src/data.json [hunk 6]
```diff
@@ -511,36 +511,109 @@
...
+                "location": {
+                    "title": "Location",
+                    "text": "London"
+                },
+                "description": {
+                    "title": "Description",
+                    "about": [
+                        "It takes a cutting edge technology stack to provide the foundation for the cutting edge journalism that the Financial Times is renowned for. Keeping our users - customers and developers - safe is paramount across our diverse technology portfolio of platforms, products and services.",
+                        "We’re looking for a skilled developer with a keen interest in security to help us deliver the tooling, practices and processes that keep our systems and applications safe across a wide platform of SaaS and PaaS, and the latest architectural concepts such as containerised microservices and serverless.",
+                        "We’re committed to equality and diversity in the tech industry, and want to encourage applications from underrepresented backgrounds. We consider part-time roles and flexible working hours to help support this.",
```
Comment (kateloschinina) [inline src/data.json]: not sure - "to help TO support this"?

Comment (ker-an) [reply src/data.json]: I have no idea - both sound write to me!

**Pull request merged**